CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+<!--
+SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+
+SPDX-License-Identifier: MIT
+-->
+
+# Aalto Grades contributing guidelines
+
+## Getting started
+
+The project uses [Node.js](https://nodejs.org/en/) as its runtime environment
+and npm to manage dependencies.
+
+Most major GNU/Linux and BSD distributions contain packages for Node.js and
+npm. You can find a list of some common distributions and their Node.js and npm
+packages at: https://nodejs.org/en/download/package-manager/
+
+Windows and macOS users can install Node.js from:
+https://nodejs.org/en/download/
+
+For more instructions, see `client/README.md` and `server/README.md`.
+
+## Bug reports
```

Comment:
This could be a bit expanded upon. Some things that should be added:
- Bug type (design or coding)
- Estimation of the severity (e.g. on a scale of 1-5)

Reply:
I expanded on it slightly and included additional sections for design issues and feature requests, though they are empty right now. We could discuss later what exactly we want to include there.